chainstate/test-suite/src/tests/chainstate_storage_tests.rs

```diff
@@ -34,7 +34,7 @@ use utxo::UtxosStorageRead;
 #[trace]
 #[case(Seed::from_entropy())]
 fn store_coin(#[case] seed: Seed) {
-    utils::concurrency::model(move || {
+    ::utils::concurrency::model(move || {
```

Comment:
When this starts happening, it's better to find a better name for the new module.